src/commandline_argv.cpp

```diff
@@ -86,8 +86,9 @@ std::vector<std::string> read_argv([[maybe_unused]] int argc, [[maybe_unused]] c
 	return win32_read_argv(flat_cmdline);
```

Comment:
this could technically be applicable to win32 as well, the function wasn't using argc tho so I was a bit skeptical about it